composer/callbacks/global_straggler_detector.py

```diff
@@ -0,0 +1,873 @@
+from megatron.core.utils import *
+
```

Comment:
Can we add a comment/code block around the part you changed in the megablocks code?